examples/redMethod/lib/redDashboard/panels.libsonnet

```diff
@@ -9,24 +9,23 @@ local g = import 'g.libsonnet';
     ]),
 
   local timeSeries = g.panel.timeSeries,
-  local defaults = timeSeries.fieldConfig.defaults,
```

Comment:
Arguably this could be seen as a regression. There was a brief period where accessing these options here was supported but have been documented to use `<panel>.standardOptions` for a while now. Considering we're in experimental state, I'd say this breaking change is fine.